dali/operators/sequence/optical_flow/turing_of/optical_flow_turing.cc

```diff
@@ -184,12 +184,13 @@ void OpticalFlowTuring::SetInitParams(dali::optical_flow::OpticalFlowParams api_
 NV_OF_EXECUTE_INPUT_PARAMS OpticalFlowTuring::GenerateExecuteInParams
         (NvOFGPUBufferHandle in_handle, NvOFGPUBufferHandle ref_handle,
          NvOFGPUBufferHandle hints_handle) {
-  NV_OF_EXECUTE_INPUT_PARAMS params;
+  NV_OF_EXECUTE_INPUT_PARAMS params = {};
   params.inputFrame = in_handle;
   params.referenceFrame = ref_handle;
   params.disableTemporalHints = of_params_.enable_temporal_hints ? NV_OF_FALSE : NV_OF_TRUE;
   params.externalHints = hints_handle;
   params.padding = 0;
+  params.padding2 = 0;
```

Comment:
Why? Now that `params` are initialized, I'd remove all of these zero assignments.

Reply:
The header says is must be set to zero, so I left the explicit assignments.

Reply:
Well, `={}` does 0-fill on a structure, so the requirement is satisfied. If coverity doesn't protest, remove it.